presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,213 @@
+===================
+LDAP Authentication
+===================
+
+Presto can be configured to enable frontend LDAP authentication over
+HTTPS for clients, such as the :ref:`cli_ldap`, or the JDBC and ODBC
+drivers. This involves user and password validation backed by LDAP.
```

Comment:
How about something like this instead, which mentions that we only support "simple" LDAP authentication mechanism:

At present, only the <i>simple</i> LDAP authentication mechanism involving user name and password is supported.